server/lib/poller.js

```diff
@@ -65,29 +65,61 @@ class Poller
         }
 
         try {
-
             // Check accounts and operations!
-            let checkAccounts = await Config.findOrCreateDefaultBooleanValue(
+            let checkAccounts = false;
+            checkAccounts = await Config.findOrCreateDefaultBooleanValue(
                 'weboob-auto-merge-accounts'
             );
-
-            log.info('Checking new operations for all accesses...');
-            if (checkAccounts) {
-                log.info('\t(will also check for accounts to merge)');
-            }
-
+        } catch (err) {
+            log.error(`Could not retrieve 'weboob-auto-merge-accounts': ${err.toString()}`);
+        }
+        // We go on even if the parameter weboob-auto-merge-accounts is
+        // not caught. By default, the merge is not done.
+        log.info('Checking new operations for all accesses...');
+        if (checkAccounts) {
+            log.info('\t(will also check for accounts to merge)');
+        }
+        try {
             let accesses = await Access.all();
             for (let access of accesses) {
                 let accountManager = new AccountManager;
-                if (checkAccounts) {
-                    await accountManager.retrieveAccountsByAccess(
-                        access,
-                        false
-                    );
+                try {
+                    // Only import if lat import did not raise a
+                    // login/parameter error
+                    if (access.canAccessBeUpdated()) {
+                        if (checkAccounts) {
+                            await accountManager.retrieveAccountsByAccess(
+                                access,
+                                false
+                            );
+                        }
+                        await accountManager.retrieveOperationsByAccess(
+                            access, cb);
```

Comment:
Wow, this is weird to read. Can't it fit on a single line? Alternatively, can you format it like this:

```
await gnagnagna.gnagnagna(
   access, cb
);
```